pkg/services/sqlstore/migrations/ualert/permissions.go

```diff
@@ -0,0 +1,162 @@
+package ualert
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/util"
+
+	"github.com/grafana/grafana/pkg/infra/metrics"
+	"github.com/grafana/grafana/pkg/models"
```

Comment:
Don't know that this it is safe to use these models / commands. As if the models change in the future, the migration will have changed.

Reply:
It makes sense; I will change it